frontend/src/components/datasets/DatasetTable.tsx

```diff
@@ -45,6 +46,17 @@ const getTableColumns = () => {
             dataIndex: "lane",
             sorter: true,
         },
+        {
+            title: "Validated",
```

Comment:
"Validated" does not really match the content of the column ... Maybe "Validation Status"